lib/shims/eq/R2015b/contains.m

```diff
@@ -0,0 +1,9 @@
+function result = contains(str, pat)
+
+% CONTAINS Determine if pattern is in strings
+% CONTAINS Introduced in R2016b
```

Comment:
Can we get a blank comment line after the H1?

Reply:
helptexts have been rewritten, and PR is updated.